buildpack.md

```diff
@@ -1049,6 +1053,19 @@ optional = false
 id = "<stack ID>"
 mixins = ["<mixin name>"]
 
+[[assets]]
+sha256 = "(required)"
+name = "(optional)"
+id   = "(required)"
+version = "(required)"
+uri = "(optional)"
+licenses = ["(optional)"]
+description = "(optional)"
+homepage = "(optional)"
+stacks = ["(required)"]
```

Comment:
did we want to make `stacks` optional here for instance jars can be stack independent.